airtable/emails.js

```diff
@@ -0,0 +1,229 @@
+// Neighbor Express Email Automation Script
+//==========================================================
+
+// This is a script that's intended to run in the Airtable Scripting Blocks environment.
+// To set it up for a city:
+
+// * Copy paste this into the Airtable Scripting Block environment for a city
+// * Update the Global and City-Specific configuration sections below
+// * Test it out by creating a test delivery and volunteer, and confirming that
+// emails send as expected.
+
+//==========================================================
+// Global configuration -- insert global Nex sendgrid proxy credentials
+//==========================================================
+// This authenticates us to the Nex sendgrid proxy, a small Zeit function
+// that forwards our request to the Sendgrid API.
+// We need to keep this secret because anyone with this credential
+// can send email on our behalf
+const SENDGRID_PROXY_TOKEN= <insert credential here>
+
+//==========================================================
+// City-specific configuration: update to match the city
+//==========================================================
+
+// For each email type, point to the Sendgrid template ID
+// Find the Sendgrid template IDs in Sendgrid UI.
+const SENDGRID_TEMPLATES = {
+    "Matched: to Volunteer": <insert template id>
+    "Matched: to Recipient": <insert template id>
+}
+
+// todo: update paterson-specific
+const REPLY_TO = {
+    name: "Neighbor Express",
+    email: "neighborexpress@gmail.com"
+}
+
+// Customize what fields we pull in from a delivery for an email template.
+// If the city has adjusted field names, they need to be updated here too.
+
+async function getEmailInfoForDelivery(delivery) {
```

Comment:
This is a sample config for Paterson, but needs to be updated to be city-specific